src/persist/zcl_abapgit_persistence_db.clas.abap

```diff
@@ -160,6 +163,16 @@ CLASS zcl_abapgit_persistence_db IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD list_by_keys.
+    FIELD-SYMBOLS: <ls_key> LIKE LINE OF it_keys.
+    LOOP AT it_keys ASSIGNING <ls_key>.
+      SELECT * FROM (c_tabname)
+      APPENDING TABLE rt_contents
+      WHERE value = <ls_key>.
```

Comment:
hmm, should `TYPE` be part of this SELECT? add as input parameter

![image](https://user-images.githubusercontent.com/5888506/135875785-3b2534f6-0861-4bcb-b403-08a145816c9d.png)

Reply:
added